aspnet/mvc/models/validation.rst

```diff
@@ -1,8 +1,196 @@
-.. include:: /../common/stub-topic.txt
+Model Validation 
+================
+By `Rachel Appel <http://github.com/rachelappel>`_
 
-|stub-icon| Model Validation
-============================
+In this article:
 
-.. include:: /../common/stub-notice.txt
+.. contents:: Sections
+  :local:
+  :depth: 1
 
-.. _issue: https://github.com/aspnet/Docs/issues/116
+Introduction to model validation
+--------------------------------
+Before an app stores data in a database, the app must validate the data. Data must be scrubbed for security, verified that it is appropriately formatted by type and size, and it must conform to your rules. Validation is necessary although it can be redundant and tedious to implement. In MVC, validation happens on both the client and server. 
+
+With data annotations in MVC apps you can validate data by customizing models so they can enforce business rules against models by applying attributes to model properties. These attributes generate validation code, thereby reducing the amount of code you must write. Fortunately, .NET has abstracted validation into the data annotations, often reducing validation to a single line of code.
+
+Data annotations
+----------------
+Data annotations are a way to configure model validation so it's similar conceptually to validation on fields in tables. This includes constraints such as assigning data types or required fields. Other types of validation include applying patterns to data to enforce business rules, such as a credit card, phone number, or email address (regular expressions). Data annotations do all this for you automatically.
```

Comment:
> Data annotations do all this for you automatically.

Not really. It's too much information for this doc but FYI:
- Users come up with a set of requirements their model must meet.
- Users annotate their models w/ data annotations detailing those requirements.
- The data annotation attributes _contain_ code enforcing the requirements.
- MVC (or WPF or whatever UI framework) ensures the annotations are checked throughout the model i.e. they call `IsValid()` on the applicable annotations.

Probably enough to say "Data annotations make enforcing these requirements much simpler and easier to use.